### PR TITLE
Feature-styling

### PR DESCRIPTION
**Pull request by maximoortelli**
### Feature-styling:

_Now that all the functionality is done, i have to add styling._

_I will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/). [Login to Zeplin](https://app.zeplin.io/login) (hint: i will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project) and open the project Bookstore CMS._

### Project Requirements:

- [x] Add any missing JSX elements to my project (without adding extra functionality).
- [x] My final design should match Zeplin's design (this includes all elements + the same styling).
- [x] My may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/routes/Navigation.js [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+    <h1 className="title-app  fontsty-normal montserrat">Bookstore CMS</h1>
+    <ul className="nav-list flex">
+      <li className="flex montserrat">
+        <Link className="link montserrat" to="/">Books</Link>
         {/* Link to the initial page */}
       </li>
       <li>
-        <Link to="/categories">Categories</Link>
+        <Link className="link" to="/categories">Categories</Link>
         {/* Link to the category page */}
```
Comment (mahbubul14) [inline src/routes/Navigation.js]: - [x] Kindly try to match your design as per the given design. The "Categories" text color should match the provided text color.

Your design             |  Provided Design
:-------------------------:|:-------------------------:
yourdesign | provided

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @maximoortelli ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [x] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/routes/Navigation.js (hunk 1 above)
Comment (maximoortelli) [reply src/routes/Navigation.js]: Done, thank you 👍😃

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @maximoortelli , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implemnting the requested change. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maximoortelli** [comment]
Thank you very much for the review @ridwanediallo, can you approve the project in Microverse? 👍😃

**Pull request merged**